### PR TITLE
Add an SNS notification plugin

### PR DESCRIPTION
**Pull request by offbyone**
It struck me that a notification engine that pumps out SNS messages might be cool to have.

### CODE REVIEW

**Comment by offbyone** [comment]
Is there anything I could do that might expedite merging this? If there is a need for documentation, tests, etc, I'm happy to add.

**Comment by liiight** [comment]
I think @gazpachoking is on vacation. Be patient, nothing else to do :-)

File: flexget/plugins/output/notify_sns.py [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    # this has to run near the end of the plugin chain, because we
+    # should notify after all other outputs.
+    @plugin.priority(0)
+    def on_task_output(self, task, config):
+        # verify that we actually support Boto 3
+        try:
+            import boto3
+        except ImportError as e:
+            log.debug("Error importing boto3: %s", e)
+            raise plugin.DependencyError("sns", "boto3", "Boto3 module required. ImportError: %s" % e)
```
Comment (gazpachoking) [inline flexget/plugins/output/notify_sns.py]: I think maybe we should do this on task start, that way it fails when we do a `--test` as well (output phase is disabled during test.)

Comment (offbyone) [inline flexget/plugins/output/notify_sns.py]: Done.

**Comment by gazpachoking** [comment]
Thanks! Want to make a wiki page?

**Comment by offbyone** [comment]
Also done :)

http://flexget.com/wiki/Plugins/sns

**Comment by liiight** [comment]
You should also edit the main Plugins wiki page and add it there as well

**Pull request merged**